lib/compat/wordpress-6.1/block-patterns/centered-footer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Footer: Centered footer.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'      => _x( 'Centered footer', 'Block pattern title' ),
+	'blockTypes' => array( 'core/template-part/footer' ),
+	'categories' => array( 'footer' ),
+	'content'    => '<!-- wp:group {"align":"full"} -->
+					<div class="wp-block-group alignfull"><!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"4em","bottom":"2em"}}},"layout":{"inherit":false}} -->
+					<div class="wp-block-group alignfull" style="padding-top:4em;padding-bottom:2em"><!-- wp:paragraph {"align":"center"} -->
+					<p class="has-text-align-center">Proudly Powered by <a href="https://wordpress.org">WordPress</a></p>
```

Comment:
Since we are using PHP in the block pattern, and the translation function is available, I  think it would be good idea to make this text and the link translatable.

Reply:
Good catch, as well as in the other patterns.

Reply:
Done in 0836d91882caaccd3a7984be38362a8d0f074b8a. What do you think?